### DISCUSSION TITLE
Error with the sync_with_roboflow

### DISCUSSION THREAD

**Discussion by Teddybiovlsi** [roboflow/supervision#795] [Q&A] [answered]
Hello, I have some problem in these code of autodistill error

when I tring to auto labeling the image everving will be fine in the below of the code,
but the final step of the autolabeling to store the image the error occur with the **ValueError: too many values to unpack (expected 5)**

this is my flowing code and error picture
appreciate for all the answers

```Python
# Before you run the Python code snippet below, run the following command:
# !pip install roboflow autodistill autodistill_grounding_dino pip install scikit-learn

from autodistill_grounding_dino import GroundingDINO
from autodistill.detection import CaptionOntology
from autodistill.helpers import sync_with_roboflow

BOX_THRESHOLD = 0.25
CAPTION_ONTOLOGY = {
    "badminton racket": "racket"
}
TEXT_THRESHOLD = 0.70

model = GroundingDINO(
    ontology=CaptionOntology(CAPTION_ONTOLOGY),
    box_threshold=BOX_THRESHOLD,
    text_threshold=TEXT_THRESHOLD
)

sync_with_roboflow(
    workspace_id=,
    workspace_url=,
    project_id =,
    batch_id = ,
    model = model
)
```
![1706344652274](https://github.com/roboflow/supervision/assets/117162287/73160560-6447-48dc-a5dd-9816c3493a75)

**Comment by SkalskiP** [comment, marked as answer]
Hi, @Teddybiovlsi! 👋🏻 Thanks for letting us know. I already made the [fix](https://github.com/roboflow/supervision/pull/798) that will be released soon. For now please use older version of supervision that most likely won't cause problems. `pip install supervision==0.17.1`
